src/test/scala/is/hail/methods/ExprSuite.scala

```diff
@@ -789,13 +789,13 @@ class ExprSuite extends SparkSuite {
       x.get(false) should contain theSameElementsAs Seq(1, 3, 5)
     }
 
-    (eval[Map[String, IndexedSeq[Int]]]("[1].tail().groupBy(k => if (k % 2 == 0) \"even\" else \"odd\")").get
+    (eval[Map[String, IndexedSeq[Int]]]("[1].filter(k => false).groupBy(k => if (k % 2 == 0) \"even\" else \"odd\")").get
```

Comment:
I'm not sure what `filter(k => false)` is doing. Is that the same as `tail()`?

Reply:
I removed head and tail, because they weren't used from Python, and they're easily replicated by slicing. I was confused by this use of tail, and assumed there was no direct way to write an empty array in the old expression language. So the filter is another way to get an empty array. But now that I try it, `[]` does work, so I really don't understand what the `tail` was doing. Anyways, I changed this to `[]`.